### PR TITLE
patch: whitelist fbis ip address

### PR DESCRIPTION
**Pull request by tinashechiraya**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 12 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 11 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 31 sec

File: django_project/minisass/settings.py [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
 INTERNAL_IPS = ['127.0.0.1']
 
+# fbis ip address will come here todo
+WHITELISTED_IP_ADDRESSES = ['195.201.33.49','159.69.33.238']
```
Comment (NyakudyaA) [inline django_project/minisass/settings.py]: This should be an env and set at runtime

**Review by NyakudyaA — CHANGES_REQUESTED** [review]
Set as ENV ip address

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 12 min

**Review by NyakudyaA — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 12 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 26 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 32 sec

**Pull request merged**